configure

```diff
@@ -675,6 +680,11 @@ def configure_node(o):
   else:
     o['variables']['node_tag'] = ''
 
+  if options.release_urlbase:
+    o['variables']['node_release_urlbase'] = options.release_urlbase
```

Comment:
you're right @thefourtheye, it shouldn't be there if it's not set, this is a leftover from a previous incarnation that needed it